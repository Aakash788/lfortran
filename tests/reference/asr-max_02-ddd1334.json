{
    "basename": "asr-max_02-ddd1334",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/max_02.f90",
    "infile_hash": "bd78accf5e71c42d049416709f9c7e4f8e68f8f78e66bb1a5a47834a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-max_02-ddd1334.stderr",
    "stderr_hash": "94450f9dc25e1fe371e42ad0eb0a9628f8c7cfaf58e0c700feb0a70c",
    "returncode": 2
}