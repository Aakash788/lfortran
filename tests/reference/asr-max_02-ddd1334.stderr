semantic error: All arguments to max0 must be of the same type and kind
 --> tests/errors/max_02.f90:4:14
  |
4 |     print *, max(y, z)
  |              ^^^^^^^^^ 
